cluster/src/main/scala/io/snappydata/impl/LeadImpl.scala

```diff
@@ -239,6 +240,22 @@ class LeadImpl extends ServerImpl with Lead
     }
   }
 
+  private def checkAuthProvider(props: Properties): Unit = {
+    var authP = props.getProperty(Attribute.AUTH_PROVIDER)
+    if (authP == null) {
+      authP = props.getProperty(Attribute.SERVER_AUTH_PROVIDER)
+    }
+    if (authP != null && !"LDAP".equalsIgnoreCase(authP)) {
+      throw new IllegalArgumentException("LDAP is the only supported auth-provider currently.")
+    } else if (authP != null && !isEnterpriseEdition()) {
```

Comment:
Doesn't really change anything, right? If it goes through the first if-block, exception is thrown anyway.